tests/test_credentials.py

```diff
@@ -99,7 +96,6 @@ def test_flow():
             resource_group="resource_group",
             workspace_name="workspace_name",
         ).get_workspace()
-        return workspace
+        assert isinstance(workspace, MagicMock)
 
-    workspace = test_flow()
-    assert isinstance(workspace, MagicMock)
+    test_flow()
```

Comment:
These were changed before prefect>=2.5.1 was released; prior to that, the tests were failing